sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/bigquery/WriteTables.java

```diff
@@ -101,27 +99,28 @@ class WriteTables<DestinationT extends @NonNull Object>
   @AutoValue
   abstract static class Result {
     abstract String getTableName();
-
+    // Downstream operations may rely on pane info which will get lost after a ReShuffle
     abstract Boolean isFirstPane();
+
+    abstract Long getPaneIndex();
```

Comment:
Why are we maintaining pane index coming out of write tables? I don't see that we're using it anywhere downstream

Reply:
will revert the WriteTable.Result changes. Originally it was just to keep WritePartition.Result and WriteTable.Result similar.